tasks/scheduler.go

```diff
@@ -1,58 +1,39 @@
-package occ
+package tasks
 
 import (
-	"errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/tendermint/tendermint/abci/types"
 	"golang.org/x/sync/errgroup"
 )
 
-var (
-	ErrReadEstimate = errors.New("multiversion store value contains estimate, cannot read, aborting")
-)
-
-// Abort contains the information for a transaction's conflict
-type Abort struct {
-	DependentTxIdx int
-	Err            error
-}
-
-func NewEstimateAbort(dependentTxIdx int) Abort {
-	return Abort{
-		DependentTxIdx: dependentTxIdx,
-		Err:            ErrReadEstimate,
-	}
-}
-
-type TaskStatus string
+type status string
 
 const (
-	TaskStatusPending   TaskStatus = "pending"
-	TaskStatusExecuted  TaskStatus = "executed"
-	TaskStatusAborted   TaskStatus = "aborted"
-	TaskStatusValidated TaskStatus = "validated"
+	// statusPending tasks are ready for execution
+	// all executing tasks are in pending state
+	statusPending status = "pending"
```

Comment:
shouldnt these be `StatusPending` so theyre public?

Reply:
at the moment tasks don't leak the scheduler so nothing else cares